components/stacks-network/src/lib.rs

```diff
@@ -48,15 +48,23 @@ where
     rt.block_on(future)
 }
 
+const BITCOIND_CHAIN_COORDINATOR_SERVICE: &str = "bitcoind-chain-coordinator-service";
+const STACKS_NODE_SERVICE: &str = "stacks-node-service";
+const STACKS_API_SERVICE: &str = "stacks-api-service";
+
+const BITCOIND_RPC_PORT: &str = "18443";
+const STACKS_NODE_RPC_PORT: &str = "20443";
+const STACKS_API_PORT: &str = "3999";
+const STACKS_API_POSTGRES_PORT: &str = "5432";
```

Comment:
Should we add a TODO here?

Reply:
Would you be in favor of removing the `const`s and just having these values directly in the map of hosts? The service names and ports should always stay the same, and the only thing that should change from one devnet to another is the namespace.

I think my preference was to have these as constants so that these numbers actually mean something to me (i.e., `18443` is the bitcoind rpc port), but I don't care too much.

Let me know how you think it's best to proceed!